src/main/java/org/cascadebot/cascadebot/music/CascadePlayer.java

```diff
@@ -204,19 +206,61 @@ public void loadFailed(FriendlyException e) {
         });
     }
 
-    public void loadPlaylist(Playlist playlist, long reqUser) {
+    public void loadPlaylist(String name, Member sender, BiConsumer<LoadPlaylistResult, List<AudioTrack>> consumer) {
```

Comment:
Is there some way we could use a return value instead of a consumer to make it consistent with save?

Reply:
Nope because loading the playlists requires a consumer.

Reply:
What do you mean down the line?

Reply:
The `loadLoadedPlaylist` method. Actually thinking about it. I need to change that